src/Akka.HealthCheck.Persistence.Tests/AkkaPersistenceLivenessProbeProviderSettingsTest.cs

```diff
@@ -25,7 +25,7 @@ public AkkaPersistenceLivenessProbeProviderSettingsTest(ITestOutputHelper helper
 }}  }";
 
 
-        [Fact]
+        [Fact(DisplayName = "AkkaPersistenceLivenessProbeProviderSettingsTest_Should_Load")]
```

Comment:
Consider giving this a human-friendly name instead - that's how it'll show up in the test run results files.